advanced/Scripts/api.sh

```diff
@@ -74,26 +80,41 @@ TestAPIAvailability() {
     fi
 }
 
-Authentication() {
-    # Try to authenticate
-    LoginAPI
+LoginAPI() {
+    # If the API URL is not set, test the availability
+    if [ -z "${API_URL}" ]; then
```

Comment:
The check for `API_URL` is superfluous, as `API_URL` is not used outside of `api.sh`

Reply:
It is used for efficiency reasons. If LoginAPI is ever used twice, we have already derived how to reach the API. We can then simply reuse this knowledge without having to inquiry again.